runners/direct-java/src/test/java/org/apache/beam/runners/direct/DirectRunnerApiSurfaceTest.java

```diff
@@ -63,6 +64,9 @@ public void testDirectRunnerApiSurface() throws Exception {
             .pruningClass(DisplayData.Builder.class)
             .pruningClass(MetricResults.class)
             .pruningClass(DirectGraphs.class)
+            .pruningClass(
+                WatermarkManager.class /* TODO: BEAM-whatever Consider moving to local-java */)
```

Comment:
file or find an existing issue and update comment.

Reply:
DDone